backend/tracim_backend/lib/core/content.py

```diff
@@ -1002,14 +1003,19 @@ def _get_all_query(
         self,
         parent_id: int = None,
         content_type_slug: str = CONTENT_TYPES.Any_SLUG,
-        workspace: Workspace = None
+        workspace: Workspace = None,
+        label:str = None,
+        order_by_properties: typing.List[typing.Union[str, QueryableAttribute]] = (),
```

Comment:
There is some problems here:

* typing say List, default value is a tuple (it should be empty list)
* we can't give an list instance here, example:

``` python
>>> def foo(a=[]):
...   a.append(1)
...   print(a)
... 
>>> foo()
[1]
>>> foo()
[1, 1]
>>> foo()
[1, 1, 1]
>>> foo()
[1, 1, 1, 1]
```

Prefer:

```
def foo(a=None):
  a = a or []  # FDV
```

FDV = Forced Default Value